prac_07/guitar.py

```diff
@@ -0,0 +1,42 @@
+"""
+CP1404/CP5632 Practical
+Do-from-scratch Exercises - Create Guitar class
+Estimate: 15 minutes
+Actual: 10 minutes
+"""
+
+
+class Guitar:
+    """Represent a Guitar object"""
+    def __init__(self, name="".title(), year=0, cost=0.0):
+        """Initialise a Guitar instance"""
+        self.name = name
+        self.year = year
+        self.cost = cost
+        self.age = 0
+
+    def __str__(self):
+        """Return string of Guitar object"""
+        return f"{self.name} ({self.year}) : ${self.cost}"
+
+    def get_age(self):
+        self.age += 2023 - self.year
+        return self.age
+
+    def is_vintage(self):
+        """Determine if the Guitar object is vintage"""
+        if (2023 - self.year) >= 50:
+            return True
+        else:
+            return False
```

Comment:
If you change your get_age method like how I suggested, you can improve this method as well by simply using
"return self.get_age() >= AGE_THRESHOLD"
This just compares if the age from the get_age method is greater than 50 (I strongly recommend you make a constant for 50 because it'll definitely be a magic number for other programmers)

Also have a look at the "Working with Booleans" section on our programming guide, it explains how you can just return the Boolean result of the calculation instead of using a if else statement :) I hope this helps